doc/lua_api.txt

```diff
@@ -971,20 +971,64 @@ There are a bunch of different looking node types.
 Look for examples in `games/minimal` or `games/minetest_game`.
 
 * `normal`
+    * A node-sized cube.
 * `airlike`
+    * Invisible, uses no texture.
 * `liquid`
+    * The cubic source node for a liquid.
 * `flowingliquid`
+    * The flowing version of a liquid, appearing with various heights and
+      slopes.
 * `glasslike`
+    * Often used for partially-transparent nodes.
+    * Only external sides of textures are visible.
 * `glasslike_framed`
+    * All face-connected nodes are drawn as one volume within a surrounding
+      frame.
+    * The frame appearence is generated from the edge pixels of the first
+      texture specified in `tiles` (preferably 16x16 pixels).
+    * The glass 'shine' (or other desired detail) on each node face is supplied
+      by the second texture specified in `tiles`.
 * `glasslike_framed_optional`
+    * This switches between the above 2 drawtypes according to the clientside
+      setting 'connected glass'.
```

Comment:
underscore (it's called connected_glass and not connected glass) 
https://github.com/minetest/minetest/blob/1d385caa06cd2f636175a1641028519af45c231b/minetest.conf.example#L432

Reply:
Ok, i was referring to the checkbox in settings menu.